apps/cookbook/src/app/examples/chart-example/examples/yearly-overview-chart-example/yearly-overview-chart-example.component.html

```diff
@@ -0,0 +1,2 @@
+<h3>Yearly overview</h3>
```

Comment:
Consider moving title to showcase/examples components to align with existing examples